packages/mdc-list/adapter.ts

```diff
@@ -75,6 +75,12 @@ export interface MDCListAdapter {
    */
   isRootFocused(): boolean;
 
+  /**
+   * @param index list item index.
+   * @return true if root element is disabled.
+   */
+  isListItemDisabled(index: number): boolean;
```

Comment:
Left a comment on the CL. Could we make this `listItemAtIndexHasClass(index: number, className: string): boolean` instead? It'll avoid having to add another method if we need to query for another conditional class on list items.

Reply:
Yes, good idea. This reduces potential interface modifications in the future. Please see updated changes.